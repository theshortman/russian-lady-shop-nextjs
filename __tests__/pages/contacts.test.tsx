import { render } from "@testing-library/react";
import ContactsPage from "@/pages/contacts";

test("renders contacts page unchanged", () => {
  const { container } = render(<ContactsPage />);
  expect(container).toMatchSnapshot();
});
