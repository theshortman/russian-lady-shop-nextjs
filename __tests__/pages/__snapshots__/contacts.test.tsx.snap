// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders contacts page unchanged 1`] = `
<div>
  <h1>
    Контакты
  </h1>
  <p>
    theshortman@protonmail.com
  </p>
</div>
`;
